### DISCUSSION TITLE
How to get rid of the “false frame” the transform tool leaves behind?

### DISCUSSION THREAD

**Discussion by 6xolotl** [keiwando/toonsquid#241] [Q&A] [answered]
This happens whenever I use the transform tool on a pixel layer, even if it’s just to do something like flip it horizontally. It’s distracting and it won’t let me draw outside of the grey frame. How do I disable it?

My strategy right now is to put the layer it affects into a folder and then rasterize it, but that takes entirely too long for how often I use the transform tool. Is this a bug or something??

![255EC383-CBE9-4874-84CF-7BF21715ECC8](https://user-images.githubusercontent.com/127806148/230564710-e1956fce-c01a-4c99-9196-8b4642b670e9.png)

**Comment by keiwando** [comment, marked as answer]
If you just want to move the pixels within the layer instead of the entire layer you need to enable the "Edit pixels" option at the top of the transform tool.

**Reply by 6xolotl** [reply]
Sorry, I saw this when you posted it but forgot to reply!! I was wondering what “edit pixels” did. Thank you for the help and sorry to bother ^^;
